pyOCD/flash/flash.py

```diff
@@ -211,16 +211,40 @@ def loadPageBuffer(self, bufferNumber, flashPtr, bytes):
         # transfer the buffer to device RAM
         self.target.writeBlockMemoryUnaligned8(self.page_buffers[bufferNumber], bytes)
 
+    def programPhrase(self, flashPtr, bytes):
+        """
+        Flash one page
+        """
+
+        # prevent security settings from locking the device
+        bytes = self.overrideSecurityBits(flashPtr, bytes)
+
+        # first transfer in RAM
+        self.target.writeBlockMemoryUnaligned8(self.begin_data, bytes)
+
+        # update core register to execute the program_page subroutine
+        result = self.callFunction(self.flash_algo['pc_program_page'], flashPtr, len(bytes), self.begin_data)
```

Comment:
Should this be callFunctionAndWait instead of callFunction, so when the call returns you know the processor is done writing?  Also is this the same as programPage?